tests/test_cookies.py

```diff
@@ -30,7 +30,7 @@ def dst(self, dt):
 class CookieResource:
 
     def on_get(self, req, resp):
-        resp.set_cookie('foo', 'bar', domain='example.com', path='/')
+        resp.set_cookie('foo', 'bar', domain='example.com', path='/', same_site=('Lex',))
```

Comment:
Are you explicitly testing an unsupported value (`'Lex'`) here?

The supported values are [`'Lax'` and `'Strict'`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie). It is a good idea to add some tests for unsupported values too, but maybe we should add a comment so the reader wouldn't have to guess whether it is a misspelling from the Falcon authors or not.